go/structs/message.go

```diff
@@ -2,8 +2,12 @@ package structs
 
 // Message defines the structure of websocket packets representing messages
 type Message struct {
-	ID      string                 `json:"id"`
-	Name    string                 `json:"name"`
-	Payload map[string]interface{} `json:"payload"`
-	Sender  string                 `json:"sender"`
+	ID         string         `json:"id"`
+	Name       string         `json:"name"`
+	Payload    MessagePayload `json:"payload"`
+	Sender     string         `json:"sender"`
+	ResponseTo string         `json:"response-to"`
```

Comment:
I think this json key is supposed to be camel case. Livescript changes the kabob case into camel case